pyaccel/optics.py

```diff
@@ -382,14 +384,13 @@ def calc_emittances(orbit, twiss, idx=0):
         acc, rin, nr_turns=nr_turns, turn_by_turn='closed', element_offset=0)
 
     if mode == 'fitting':
-        r = _np.dstack([rin[None, :, None], rout])
-        ax, bx = fitEllipse(r[0][0], r[0][1])
-        ay, by = fitEllipse(r[0][2], r[0][3])
+        ax, bx = fitEllipse(rout[0], rout[1])
```

Comment:
three-letter pylint rule in this section.

Reply:
this module has an infinity of pylint violations... :tired_face: 
I'll fix this one you've pointed out!

Reply:
I know! I only commented the sections modified in this PR. later I'll create a _Standardize_ PRs for _pyaccel_ and _apsuite_ and submit them to you all.

Reply:
done :face_with_head_bandage: